schema.sql

```diff
@@ -7,4 +7,7 @@ CREATE TABLE animals(
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL
-);
```

Comment:
Although I am highlighting this file, this change should be done to the _**README.md**_ file.

- [OPTIONAL] Kindly consider removing this section of your README that is captured in the screenshot below:

![image](https://github.com/Abdulwhbhussain/Vet-clinic/assets/77106962/45b59dd2-0188-448d-9988-06540737a40f)

You already have details of your social media links made available in the first entries. 👍 😎

Reply:
Implemented